R-package/tests/testthat/test_model.R

```diff
@@ -172,6 +172,7 @@ test_that("Fine-tune", {
 })                                       
 
 test_that("Matrix Factorization", {
+  skip("Disabled due to an unavailible http server.  Tracked here: http://bit.ly/2qybsto")
```

Comment:
Please link the issue directly

Reply:
I'm not opposed, but can you provide some motivation for the change?  I shortened it to avoid line limits and maintain readability.